tests/docs/source/autoconfigurable/specified_members.rst

```diff
@@ -0,0 +1,15 @@
+autoconfigurable - specified members
+====================================
+
+The ``members`` option with specified members should only present the specified
+members.
+
+In this test we list the member ``method`` and ``trait_nohelp``. One may expect
+no members besides these to show up, but we present them and all trait members
+still, see `Issue27`_.
+
+.. _Issue27: https://github.com/jupyterhub/autodoc-traits/issues/27
+
+.. autoconfigurable:: sample_module.SampleConfigurable
+   :noindex:
+   :members: method,trait_nohelp
```

Comment:
`trait_nohelp` is not included. Does the test cover this?

![Screenshot 2022-12-15 at 12 54 26](https://user-images.githubusercontent.com/151929/207852952-4011459c-65f2-40c9-a4e1-b21dd6e2b7d9.png)

Reply:
Hmmm... It is for me.

Here is the segment of code testing that it is: https://github.com/consideRatio/autodoc-traits/blob/pr/refresh-and-add-tests/tests/test_autodoc_traits.py#L76-L85

Here is a screenshot from my computer using sphinx 4, tested to do the same with sphinx 5 and sphinx 6beta3 also: 
![image](https://user-images.githubusercontent.com/3837114/207861147-b58b0b98-0ded-4df4-9256-1f34b5ed0f53.png)

Reply:
Huh, ok. Must be something weird in my setup.